js/deribit.js

```diff
@@ -2074,6 +2080,135 @@ module.exports = class deribit extends Exchange {
         return result;
     }
 
+    async fetchTransfers (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (code === undefined) {
+            throw new ArgumentsRequired (this.id + ' transfer() requires a currency code argument');
+        }
```

Comment:
I suggest that we throw early in such cases – if a required code or symbol is missing we should raise an exception even before loading the markets (where possible).

```suggestion
        if (code === undefined) {
            throw new ArgumentsRequired (this.id + ' transfer() requires a currency code argument');
        }
        await this.loadMarkets ();
```